entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+
+mkdir -p /metasource_db
+
+# If database is empty download it
+
+if [ -z "$(ls -A /metasource_db)" ]; then
+    echo "Database not found, downloading..."
+    mkdir -p /tmp/metasource_db
+    ./metasource-cli -location /tmp/metasource_db database
+    cp -r /tmp/metasource_db/* /metasource_db/
+    rm -rf /tmp/metasource_db
+fi
```

Comment:
Instead of this, may I suggest that you leave the entrypoint of the container without mentioning a specific subcommand and that way, people can bend it to fill the requirements of both the `database` and `dispense` subcommands?